x-pack/plugins/telemetry_collection_xpack/server/telemetry_collection/get_stats_with_xpack.test.ts

```diff
@@ -53,34 +53,43 @@ const mockUsageCollection = (kibanaUsage = kibana) => ({
   toObject: (data: any) => data,
 });
 
+function clearEsClientMock(esClient: any) {
+  esClient.xpack.usage.mockClear();
+  esClient.license.get.mockClear();
+  esClient.nodes.usage.mockClear();
+  esClient.info.mockClear();
+}
```

Comment:
I think there's no need to clear the `esClient` mocks because we create new `esClient`s for each test.